test/process_wrapper/fake_rustc.rs

```diff
@@ -0,0 +1,7 @@
+// This binary mocks the output of rustc when run with --error-format=json and --json=artifacts.
```

Comment:
Oddly, `rustdoc` reads `//!` when it's at the top of a file to describe the module. But one more nit.
```suggestion
//! This binary mocks the output of rustc when run with `--error-format=json` and `--json=artifacts`.

```